react/Viewer/Viewer.jsx

```diff
@@ -118,12 +119,18 @@ Viewer.propTypes = {
   showNavigation: PropTypes.bool,
   /** A render prop that is called when a file can't be displayed */
   renderFallbackExtraContent: PropTypes.func,
-  /** Used to open an Only Office file */
-  onlyOfficeProps: PropTypes.shape({
-    /** Whether Only Office is enabled on the server */
-    isEnabled: PropTypes.bool,
-    /** To open the Only Office file */
-    opener: PropTypes.func
+  /* Props passed to components with the same name */
+  componentsProps: PropTypes.shape({
+    /** Used to open an Only Office file */
+    OnlyOfficeViewer: PropTypes.shape({
+      /** Whether Only Office is enabled on the server */
+      isEnabled: PropTypes.bool,
+      /** To open the Only Office file */
+      opener: PropTypes.func
+    }),
+    ViewerControls: PropTypes.shape({
+      displayContentTop: PropTypes.bool
```

Comment:
juste pour être raccord avec https://github.com/cozy/cozy-guidelines/#naming-of-functions 

isSomething ou hasSomething ;) (on peut aussi imaginer shouldSomething)